src/ca11y/index.js

```diff
@@ -235,6 +361,17 @@ class Ca11y {
   }
 }
 
+/**
+ * Ca11y.pickers is used to globally track instances of Ca11y within a give page.
+ **/
 Ca11y.pickers = []
 
+/**
+ * Ca11y.init is used to instantiate multiple datepickers on a page simultaneously.
+ **/
+Ca11y.init = function(selector, options) {
+  const datePickers = Array.from(document.querySelectorAll(selector) || 0)
```

Comment:
Really rough... `Array.from` is supported in Firefox and Chrome, which are the only test targets we typically set up for Karma. We should explore setting up Browserstack to catch IE